api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreProperties.java

```diff
@@ -29,9 +29,14 @@ public void setApplicationContext(ApplicationContext applicationContext) {
         this.applicationContext = applicationContext;
     }
 
-    public Integer getOpenMrsIdentifierSourceId() {
+	public Integer getOpenMrsIdentifierSourceId() {
         String propertyName = RegistrationCoreConstants.GP_OPENMRS_IDENTIFIER_SOURCE_ID;
-        return getIntegerProperty(propertyName);
+		return getIntegerProperty(propertyName);
```

Comment:
Did anything change here?

Reply:
Formatting caused as a result of:  https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse. Planning to add [this](https://github.com/mekomsolutions/openmrs-module-initializer/blob/master/pom.xml#L52-L83) to generally improve on the formatting.